src/js/App/Header/HeaderTests/UserToggle.test.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable quotes */
+/* eslint-disable camelcase */
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import ConnectedUserToggle, { UserToggle } from '../UserToggle';
+import { Provider } from 'react-redux';
+
+describe('UserToggle', ()=>{
+    it('should render correctly with isSmall false', () =>{
+        const props = {
+            isOpen: false,
+            account: {
+                number: 'someNumber',
+                name: 'someName'
+            },
+            isSmall: false,
+            extraItems: []
+        };
+        const mockSelect = jest.fn();
+        const wrapper = shallow(
+            <UserToggle {...props} onSelect = { mockSelect } />
+        );
+        expect(toJson(wrapper)).toMatchSnapshot();
+        wrapper.find("[widget-type='InsightsOverflowActions']").simulate('select');
```

Comment:
Same as above, no need to disable quotes, you can use backticks ` also known as string literals (template string)

Reply:
done